x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -2,63 +2,88 @@
-| `authentication_success`          | | | Logged when a user successfully authenticates.
-| `authentication_failed`           | | | Logged when the authentication token cannot be
-                                          matched to a known user.
-| `realm_authentication_failed`     | | | Logged for every realm that fails to present a valid
-                                          authentication token. `<realm>` represents the
-                                          realm type.
-| `access_denied`                   | | | Logged when an authenticated user attempts to execute
-                                          an action they do not have the necessary
-                                          <<security-privileges,privilege>> to perform.
-| `access_granted`                  | | | Logged when an authenticated user attempts to execute
-                                          an action they have the necessary privilege to perform.
-                                          When the `system_access_granted` event is included, all system
-                                          (internal) actions are also logged. The default setting does
-                                          not log system actions to avoid cluttering the logs.
-| `run_as_granted`                  | | | Logged when an authenticated user attempts to <<run-as-privilege, run as>>
-                                          another user that they have the necessary privileges to do.
-| `run_as_denied`                   | | | Logged when an authenticated user attempts to <<run-as-privilege, run as>>
-                                          another user action they do not have the necessary
-                                          <<security-privileges,privilege>> to do so.
-| `tampered_request`                | | | Logged when the {security-features} detect that the request has
-                                          been tampered with. Typically relates to `search/scroll`
-                                          requests when the scroll ID is believed to have been
-                                          tampered with.
-| `connection_granted`              | | | Logged when an incoming TCP connection passes the
-                                          <<ip-filtering, IP Filter>> for a specific
-                                          profile.
-| `connection_denied`               | | | Logged when an incoming TCP connection does not pass the
-                                          <<ip-filtering, IP Filter>> for a specific
-                                          profile.
+When you are <<enable-audit-logging,auditing security events>>, a single client request
+might generate multiple audit events, across multiple cluster nodes. The common
+`request.id` attribute can be used to correlate the associated events.
+
+The following is an exhaustive list of the events that can be generated:
+
+|======
```

Comment:
It would be nice to have a header row that explains the purpose of each column. For example:
```suggestion
[options="header"]
|======
| Event type
| Description
| Example
```
... in that case, you likely don't have to put "Example" in the third column's entry every time.

Reply:
Yes! Thanks for noticing, I forgot to ask, I tried to find an example for the table header, but ultimately bailed out.

One nit: `Event type` -> `Event action` . The thing is that these values here are printed next to the `event.action` attribute, and that there is another attribute `event.type` (with values `rest`, `transport`, `ip_filter` and `security_config_change`).
Traditionally we've referred to these values as types, but given the above, I think this is confusing. I've opted for the "event kind" terminology, because it is a synonym for "type" and it is not present in the event schema.